vcx/libvcx/src/v3/handlers/issuance/holder.rs

```diff
@@ -199,6 +199,16 @@ impl HolderSM {
             _ => Err(VcxError::from_msg(VcxErrorKind::NotReady, "Cannot get credential: Credential Issuance is not finished yet"))
         }
     }
+
+    pub fn delete_credential(&self) -> VcxResult<()> {
+        match self.state {
+            HolderState::Finished(ref state) => {
+                let cred_id = state.cred_id.clone().ok_or(VcxError::from_msg(VcxErrorKind::InvalidState, "Cannot get credential: Credential Id not found"))?;
+                _delete_credential(&cred_id)
+            }
+            _ => Err(VcxError::from_msg(VcxErrorKind::NotReady, "Cannot delete credential: Credential Issuance is not finished yet"))
```

Comment:
Maybe we should add `ErrorKind` and `ErrorCode` for these errors, not sure.

Reply:
I think we can reuse the existing one NotReady (Object not ready for specified action) because it is used in the get_credential (https://github.com/hyperledger/indy-sdk/blob/6ecf3dded79fde74206ec0939d2dda179784768c/vcx/libvcx/src/v3/handlers/issuance/holder.rs#L199) that has the same context.